src/Database/Database.php

```diff
@@ -535,6 +535,227 @@ public function createAttribute(string $collection, string $id, string $type, in
         return $attribute;
     }
 
+    /**
+     * Update attribute metadata. Utility method for update attribute methods.
+     * 
+     * @param string $collection
+     * @param string $id
+     * @param string $key Metadata key to update
+     * @param callable $updateCallback method that recieves document, and returns it with changes applied
+     * 
+     * @return Document
+     */
+    private function updateAttributeMeta(string $collection, string $id, callable $updateCallback): void
+    {
+        // Load
+        $collection = $this->getCollection($collection);
+
+        $attributes = $collection->getAttribute('attributes', []);
+
+        $attributeIndex = \array_search($id, \array_map(fn($attribute) => $attribute['$id'], $attributes));
+
+        if($attributeIndex === false) {
+            throw new Exception('Attribute not found');
+        }
+
+        $attribute = $attributes[$attributeIndex];
+
+        // Execute update from callback
+        call_user_func($updateCallback, $attribute);
+
+        // Save
+        $attributes[$attributeIndex] = $attribute;
```

Comment:
```suggestion
        // Execute update from callback
        call_user_func($updateCallback, $attributes[$attributeIndex]);
```

Not sure if this does the same? 🤔

Reply:
It does, still works as a refference. Test passing.